### ISSUE TITLE
(RFC) Create a separate InfluxDB "Cloud" version

### ISSUE THREAD

**Issue by sanderson** [influxdata/docs-v2#1374]
Right now, the InfluxDB OSS and Cloud docs are grouped together, but Cloud functionality releases ahead of OSS releases. These functionality gaps will continue to widen as we get into the OSS GA release cycle.

## Proposal
Create a new "Cloud" version for InfluxDB that will always live at the top of the InfluxDB version list:

![image](https://user-images.githubusercontent.com/181484/91592519-e8f54b80-e91b-11ea-90bc-31e010533279.png)

In the directory structure, the cloud content would be a separate directory in `content/influxdb/`

```
content/influxdb/
  - cloud
  - v1.3
  - v1.4
  ...
```

Whenever we cut a new OSS release, we would duplicate the `cloud` directory and remove all the cloud-specific content.

##### Side effects
- We could do away with the product tags in the left nav and at the top of pages.
- Duplicating the Flux docs in cloud and in each OSS release will result in a lot of unnecessary duplicate pages (and effort). This change may require separating the Flux reference documentation into its own content directory.
- Cloud would be treated as the canonical source for all duplicate content, so all OSS pages would need a canonical link to their cloud counterpart. We would also need logic that detects if an OSS page has a cloud counterpart and, if not, set the canonical link to itself.

**Comment by timhallinflux** [comment]
Cloud is not necessarily versioned... as it operates more on a continuous delivery model.

**Comment by sanderson** [comment]
@timhallinflux Right. I'm using "version" loosely here. Structurally it would act like a version, but as the things are added to to Cloud, we would immediately document and deploy those changes (in the `cloud` directory). The cloud "version" would always be up to date with the most recent updates. Whenever we cut a new OSS release, we would just copy the cloud directory as a new `v2.x` OSS version release and remove any cloud-specific content from that new directory.

**Comment by timhallinflux** [comment]
right, but we went down this path before.  So, the question is... is Cloud a replacement for OSS?  (i.e. are these fungible products??).  They are not in my view.  While they have functionality that is largely similar -- they should begin to diverge here soon.  OSS good for local development...  OSS deployed at the edge (and register it with Cloud - in the future).

So, I think you are right, we have approached the tipping point where splitting OSS from Cloud makes sense. It feels like GA is that point. Also, there is a vote for splitting out the CLI as well....since it works across both.
